oio/common/client.py

```diff
@@ -52,5 +52,11 @@ def _direct_request(self, method, url, session=None, headers=None,
             headers["X-oio-action-mode"] = "autocreate"
             kwargs = kwargs.copy()
             kwargs.pop("autocreate")
+        if kwargs.get("append"):
```

Comment:
Append mode should not be selected by a header, but preferably by a query string parameter.